lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -211,14 +224,17 @@ def __init__(self, ticksize, tick_out=False, *, axis=None, **kwargs):
                 kwargs["markeredgewidth"] = "auto"
 
         Line2D.__init__(self, [0.], [0.], **kwargs)
-        AttributeCopier.__init__(self, self._axis, klass=Line2D)
+        self._klass = Line2D  # Only during the AttributeCopier deprecation.
         self.set_snap(True)
 
     def get_ref_artist(self):
```

Comment:
Can we make this private (`_get_ref_artist`)? I don't think ref artists need to be part of  the public API.

Also please add a docstring such as:

> Return an underlying artist, that actually defines the properties (e.g. color) of self.

Reply:
Honestly I wouldn't bother with the deprecation; it's not as if there's any plan to actually change that method.
Added the docstring.

Reply:
You've deprecated the whole `AttributeCopier`, which originally introduced `get_ref_artist`. It would only be consquent to remove `get_ref_artist` as well from the public API. This is basically foremost for cleaning up the public API. ref artists are an implementation detail that should not be publicly visible. Users should not be bothered with this (neither in the docs nor in parameter completion etc.). Future ability to change is only the weaker motivation.